### PR TITLE
SPIDER: Animal Care and Control Commission

### PR DESCRIPTION
**Pull request by palewire**
For #146

### CODE REVIEW

File: documenters_aggregator/spiders/accc.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
+"""
+All spiders should yield data shaped according to the Open Civic Data
+specification (http://docs.opencivicdata.org/en/latest/data/event.html).
+"""
+import scrapy
+from datetime import datetime
```
Comment (eads) [inline documenters_aggregator/spiders/accc.py]: Pyflakes says this is imported but unused.

File: tests/test_accc.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import pytest
```
Comment (eads) [inline tests/test_accc.py]: Imported by unused, per pyflakes.

File: documenters_aggregator/spiders/accc.py [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+                'all_day': self._parse_all_day(text),
+                'status': self._parse_status(text),
+                'location': self._parse_location(text),
+            }
+
+    def _parse_id(self, item):
+        """
+        Calulate ID. ID must be unique within the data source being scraped.
+        """
+        return None
```
Comment (eads) [inline documenters_aggregator/spiders/accc.py]: We need an ID for the event. It just has to be unique within the universe of data the spider is collecting, so you could just slugify name and date/time as long as they never have more than one meeting at the same time. Or see if there's an ID in the DOM.

**Comment by palewire** [comment]
I think the changes you've requested have been pushed.

**Comment by eads** [comment]
kitten cop

**Pull request merged**